Netezos/Forging/Models/Operations/Content/SrRefuteContent.cs

```diff
@@ -0,0 +1,150 @@
+using System.Text.Json.Serialization;
+
+namespace Netezos.Forging.Models;
+
+public class SrRefuteContent : ManagerOperationContent
+{
+    [JsonPropertyName("kind")]
+    public override string Kind => "smart_rollup_refute";
+
+    [JsonPropertyName("rollup")]
+    public string Rollup { get; set; }
+    
+    [JsonPropertyName("opponent")]
+    public string Opponent { get; set; }
+    
+    [JsonPropertyName("refutation")]
+    public Refutation Refutation { get; set; }
+}
+
+[JsonConverter(typeof(RefutationConverter))]
+public abstract class Refutation
+{
+    [JsonPropertyName("refutation_kind")]
+    public abstract string RefutationKind { get; }
+}
+
+public class RefutationStart : Refutation
+{
+    [JsonPropertyName("refutation_kind")] 
+    public override string RefutationKind => "start";
+    
+    [JsonPropertyName("player_commitment_hash")]
+    public string PlayerCommitmentHash { get; set; }
+    
+    [JsonPropertyName("opponent_commitment_hash")]
+    public string OpponentCommitmentHash { get; set; }
+}
+
+public class RefutationDissectionMove : Refutation
+{
+    [JsonPropertyName("refutation_kind")] 
+    public override string RefutationKind => "move";
+    
+    [JsonPropertyName("choice")]
+    [JsonConverter(typeof(Int64StringConverter))]
+    public long Choice { get; set; }
+    
+    [JsonPropertyName("step")]
+    public List<Dissection> Step { get; set; }
+}
+
+public class RefutationProofMove : Refutation
+{
+    [JsonPropertyName("refutation_kind")] 
+    public override string RefutationKind => "move";
+    
+    [JsonPropertyName("choice")]
+    [JsonConverter(typeof(Int64StringConverter))]
+    public long Choice { get; set; }
+    
+    [JsonPropertyName("step")]
+    public ProofStep Step { get; set; }
+}
+
+public class Dissection
+{
+    [JsonPropertyName("state")]
+    public string State { get; set; }
+    
+    [JsonPropertyName("tick")]
+    [JsonConverter(typeof(Int64StringConverter))]
+    public long Tick { get; set; }
+}
+
+public class ProofStep
+{
+    [JsonPropertyName("pvm_step")]
+    public string PvmStep { get; set; }
+    
+    [JsonPropertyName("input_proof")]
+    public InputProof InputProof { get; set;}
```

Comment:
InputProof is nullable, see https://tezos.gitlab.io/shell/p2p_api.html#proof-tag-1